docs/ops/logical/LogicalNot_1.md

```diff
@@ -6,13 +6,17 @@
 
 **Short description**: *LogicalNot* performs element-wise logical negation operation with given tensor.
 
-**Attributes**:
+**Detailed description**: *LogicalNot* performs element-wise logical negation operation with given tensor, based on the following mathematical formula:
 
-    No attributes available.
+\f[
+a_{i} = not(a_{i})
+\f]
```

Comment:
```suggestion
\f[
a_{i} = \lnot a_{i}
\f]
```
I think this might look a little bit better:
![image](https://user-images.githubusercontent.com/39924818/128502476-fe99bac9-e48d-46e6-b2ba-c8ab4829a300.png)